_infdemos/plyranges-workshop.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Intro to plyranges (Bioconductor)
+author: Chao-Jen Wong
+last_modified_at: 2018-08-09
+---
+
+## Getting started
+`plyranges` is a Bioconductor package that offers `dplyr`-style operations on the `GRanges` class.
+
+- [Workshop Material](https://bioconductor.github.io/BiocWorkshops/fluent-genomic-data-analysis-with-plyranges.html)
+
+## Setup
+Require R >= 3.5.0
+
+```{r}
+install.packages("BiocManager")
+library(BiocManager)
+install(c("plyranges", "AnnotationHub", "airway"))
+```
+
+
+If you intend to work via a terminal on rhino, `ml` R and Rstudio.  
+
+Connect to Rhino.
+```{bash}
+ssh -X HutchID@rhino
+```
+
+Load modules and open an RStudio window.
+```
+ml R/3.5.0-foss-2016b-fh1
+ml rstudio/1.1.383
+rstudio &
+```
+
+
+In Rstudio, load the library.
+
+```{r}
+suppressPackageStartupMessages(library(plyranges, quietly=TRUE))
+```
+
+## GRanges
+__Components__:
+
+- seqname, ranges, strands
+
+- metadata columns: annotation, covariates (right side of the dish line)
+
+```{r}
+
+gr <- GRanges(seqnames="VI",
+
+              IRanges(start=c(3322, 3030, 1437,  5066, 6426, 836),
+                      end = c(3846, 3338, 2615, 5521, 7565, 1363)),
+                      strand = c("-", "-", "-", "+", "+", "+"))
+mcols(gr) <-
+  DataFrame(gene_id=c("YFL064C", "YFL065C", "YFL066C",
+                      "YFL063W", "YFL062W", "YFL067W"))
+gr
+```
+
```

Comment:
Need to add the results of running the code chunk. What I need to do it converting *.Rmd to *.md that obtains all the cold chunk as well as the results.